src/resources/i18n/translations/en.json

```diff
@@ -391,13 +391,13 @@
   "Last": "Last",
   "Latest": "Latest",
   "Latest episode": "Latest episode",
-  "Launch screen": "Launch screen",
+  "Launch screen": "Main view",
   "Leaving App": "Leaving…",
   "License": "Copylefted libre license",
   "Light Mode": "Light",
   "Limit": "Limit",
-  "Limit the number of downloaded episodes for each podcast by default": "Limit downloaded episodes for each podcast.",
-  "Limit the number of downloaded epiosdes explanation": "Limit the number of downloaded episodes from this podcast on your device. After the download limit is exceeded, the oldest episode will be automatically deleted.",
+  "Limit the number of downloaded episodes for each podcast by default": "Individual episode-limit.",
```

Comment:
```suggestion
  "Limit the number of downloaded episodes for each podcast by default": "Individual episode-limits.",
```

Reply:
This text goes next to an on/off toggle, so I think it needs to be an on/off verb like "Limit downloaded episodes for each podcast". This PR is getting kind of crazy and scattered with my comments, so I'm going approve or skip much of these, and we can revisit in a future PR.

![IMG_4161A4C3C55E-1](https://user-images.githubusercontent.com/2262919/205472346-eca00dc8-e952-4cc1-932b-3fffd39671bf.jpeg)

Reply:
Turn on/off is implied by it being a toggle
```suggestion
  "Limit the number of downloaded episodes for each podcast by default": "Episode-specific limits",
```
Are we revisiting your comments, or the PR?
https://www.yewtu.be/watch?v=WHmpsEonrFo